docs/meetings/2020/TechArea20200330.md

```diff
@@ -38,7 +38,7 @@
     -   AI (Diego): Add option to enable the XCache data integrity script for CMS and ATLAS XCache (SOFTWARE-4026)
     -   AI (Carl): Move OASIS manager login credential information to CILogon (SOFTWARE-3947)
     -   AI (BrianL): Fix-up email templates and confusing web page for the COManage self-registration process
-    -   AI (Edgar): Try out the [self-registration process](https://osg-htc.org/technology/policy/comanage-instructions-user/)
+    -   AI (Edgar): Try out the [self-registration process](https://osg-htc.org/docs/common/contact-registration)
```

Comment:
It seems odd that we have to update links in oooold meeting notes retroactively -- it's almost like they should point to an archived link of that page at the time of the meeting.  But i don't have a strong opinion here.

Reply:
Yeah, I'm not really consistent about what I do in these cases. Sometimes I make them plain text instead of links but that's a good point